src/ui/components/Transcript/TranscriptItem/TranscriptItem.css

```diff
@@ -74,7 +74,7 @@
 
 .event-timestamp {
   font-size: 12px;
-  background: var(--grey-20);
+  background: var(--gray-20);
```

Comment:
This one's really meant to be var(--grey-20). There's some gray/grey conflict with the naming schemes :)

Reply:
Oh! Thanks!